pkgs/development/python-modules/openmm/default.nix

```diff
@@ -0,0 +1,86 @@
+{ buildPythonPackage
+, lib
+, fetchFromGitHub
+, cmake
+, gfortran
+, fftwSinglePrec
+, doxygen
+, ocl-icd
+, opencl-headers
+, swig
+, cython
+, numpy
+, cudatoolkit
+, clang
+, enableClang ? true
+, enableCuda ? false
```

Comment:
We shouldn't give pythonPackages packages such options that the user needs to change.

Reply:
There is no **need** to change. I think the defaults are reasonable. And such config options are used for the tensorflow and pytorch packages as well?

Reply:
OpenMM and its C++, C and Fortran bindings now live in all-packages. Flags to control whether to enable CUDA (false by default), OpenCL (false by default), CLang (true by default) and Python bindings are set. The defaults are reasonable IMHO and pass all tests.